apidump/apidump.go

```diff
@@ -470,18 +472,25 @@ func (a *apidump) TelemetryWorker(done <-chan struct{}) {
 	if a.TelemetryInterval > 0 {
 		ticker := time.NewTicker(time.Duration(a.TelemetryInterval) * time.Second)
 
+		lastReport := time.Now()
 		for {
 			select {
 			case <-done:
 				return
 			case now := <-ticker.C:
-				duration := int(now.Sub(a.startTime) / time.Second)
-				a.SendPacketTelemetry(duration)
+				observationDuration := int(now.Sub(a.startTime) / time.Second)
+				windowStart := lastReport
+				windowDuration := int(now.Sub(windowStart) / time.Second)
```

Comment:
Is rounding down the appropriate thing here, or should we round to the nearest second?